onnxruntime/core/optimizer/nhwc_transformer.cc

```diff
@@ -248,6 +285,8 @@ void NhwcTransformerImpl::Transform(Node& node) {
     TransformQLinearActivation(node);
   } else if (graph_utils::IsSupportedOptypeVersionAndDomain(node, "QLinearGlobalAveragePool", {1}, kMSDomain)) {
     TransformQLinearGlobalAveragePool(node);
+  } else if (graph_utils::IsSupportedOptypeVersionAndDomain(node, "Split", {2, 11, 13})) {
```

Comment:
Need to add a test case to nhwc_transformer_test.cc.

Reply:
Yes. Will add soon